Microsoft.Toolkit.Uwp.Services/Exceptions/RequestFailedException.cs

```diff
@@ -44,13 +43,23 @@ public RequestFailedException(string message)
         /// </summary>
         /// <param name="statusCode">Failure status code.</param>
         /// <param name="reason">Failure reason.</param>
-        public RequestFailedException(HttpStatusCode statusCode, string reason)
+        public RequestFailedException(Windows.Web.Http.HttpStatusCode statusCode, string reason)
```

Comment:
That is the old overload that accepts the `Windows.Web.Http.HttpStatusCode`, there is a new overload that explicitly taking in the `System.Net.HttpStatusCode ` right down below. I didn't delete this one because somebody might be using it with the `Windows.Web.Http.HttpClient` later. Explicit type is also required due to the similarity in name of the type.